aws/cloudformation/components/codeprojects.yml.erb

```diff
@@ -0,0 +1,44 @@
+<%- # Note this file must be indented one level, as it is a child of 'Resources'. %>
+<%- unless rack_env?(:adhoc) -%>
```

Comment:
I don't like this. I'd rather do something like `['staging', 'test', 'production'].include? environment`

Haven't found an existing example of that, and there are a few ways to implement it.

Reply:
Why do we exclude code projects.org from our adhoc environments? Is it because we don't have a base domain name to use (`dev-codeprojects.org`)?

Reply:
I just remembered ... we want to support `rack_env == 'development'`

Reply:
I believe you can do `if rack_env?(:staging, :test, :production)`